File: c/consumo.c
#include <stdio.h>

int main(){
    int distancia;
    double gasto, consumo;

    printf("Distancia percorrida: ");
    scanf("%d", &distancia);

    printf("Combustivel gasto: ");
    scanf("%lf", &gasto);

    consumo = distancia/gasto;

	printf("Consumo medio = %.3lf\n", consumo);

	return 0;
}

File: java/consumo.java
import java.util.Locale;
import java.util.Scanner;

public class consumo {

	public static void main(String[] args) {

		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		int distancia;
	    double gasto, consumo;

	    System.out.print("Distancia percorrida: ");
	    distancia = sc.nextInt();

	    System.out.print("Combustivel gasto: ");
	    gasto = sc.nextDouble();

	    consumo = distancia/gasto;

		System.out.printf("Consumo medio = %.3f\n", consumo);

		sc.close();
	}
}
